src/backend/web/authentication.js

```diff
@@ -105,19 +87,21 @@ function init(passport) {
     function(profile, done) {
       // TODO: we can probably pick off the user data we actually need here...
       if (!profile) {
-        const error = new Error('SAML Strategy verify callback missing profile');
+        const error = new Error('SAML Strategy verify callback missing user profile');
         logger.error({ error });
         done(error);
       } else {
-        done(null, profile);
+        // We only use the displayname, emailaddress, and nameID (hashed, for use in our db)
+        done(null, {
```

Comment:
How come this is moved to the strategy opposed to the serialization?

Reply:
I did some more reading, and I gave you incorrect advice.  Where I moved it to, when the "profile" data comes back from the SAML response, is where you're supposed to create your internal user object.  The serialize/deserialize functions are where you'd deal with getting data in/out of a database.  For example, you might store the nameID on the session, and then go get it from your database in here.  We don't need to do that, so we can pass the object along untouched.